pages/docs/dev-docs/release-strategy.md

```diff
@@ -38,6 +38,19 @@ permalink: dev-docs-release-strategy.html
 5. Make sure everything is working
 6. Release R
 
+## Release schedule
+
+The core library repository follows semantic versioning.
+
+We group together breaking changes and release a new breaking version as needed, but not more frequently than once every two years (aiming for longer). This aims to give time and motivation to users to adopt new versions, and spreads the effort of preparing a release over a longer period of time. There is no fixed schedule for major releases, and we take time to ensure that no further breaking changes are needed soon after.
```

Comment:
After discussing with @davidscn, I am removing the explicit number of years, as the exact frequency is not what is rigidly decided upon:

```suggestion
We group together breaking changes and release a new breaking (major) version as needed. There is no fixed schedule for major releases, and we take time to ensure that no further breaking changes are needed soon after. This aims to give time and motivation to users to adopt new versions, and spreads the effort of preparing a release over a longer period of time.
```